static/app/components/teamSelector.tsx

```diff
@@ -136,8 +139,8 @@ function TeamSelector(props: Props) {
   // TODO(ts) This type could be improved when react-select types are better.
   const selectRef = useRef<any>(null);
 
-  const canCreateTeam = organization.access.includes('project:admin');
-  const canAddTeam = organization.access.includes('project:write');
+  const canCreateTeam = organization?.access?.includes('project:admin') ?? false;
+  const canAddTeam = organization?.access?.includes('project:write') ?? false;
```

Comment:
This should be enough.
```suggestion
  const canCreateTeam = organization?.access.includes('project:admin') ?? false;
  const canAddTeam = organization?.access.includes('project:write') ?? false;
```